tests/gordo_components/builder/test_mlflow_utils.py

```diff
@@ -14,143 +15,213 @@
 def metadata():
     """Example metadata for testing"""
     return {
-        "user-defined": {},
-        "name": "model-name",
+        "user-defined": {
+            "global-metadata": {},
+            "machine-metadata": {
+                "information": "Some sweet information about the model"
+            },
+        },
+        "name": "crazy-sweet-name",
         "dataset": {
             "tag_list": [
-                SensorTag(name="Tag 1", asset=None),
-                SensorTag(name="Tag 2", asset=None),
+                SensorTag(name="Tag-1", asset="asgb"),
+                SensorTag(name="Tag-2", asset="asgb"),
             ],
             "target_tag_list": [
-                SensorTag(name="Tag 1", asset=None),
-                SensorTag(name="Tag 2", asset=None),
+                SensorTag(name="Tag-3", asset="asgb"),
+                SensorTag(name="Tag-4", asset="asgb"),
             ],
-            "train_start_date": datetime(2017, 12, 25, 6, 0, tzinfo=UTC),
-            "train_end_date": datetime(2017, 12, 30, 6, 0, tzinfo=UTC),
+            "train_start_date": datetime.datetime(2019, 1, 1, 0, 0, tzinfo=tzutc()),
+            "train_end_date": datetime.datetime(2019, 3, 1, 0, 0, tzinfo=tzutc()),
             "resolution": "10T",
             "filter": "",
             "row_filter_buffer_size": 0,
             "data_provider": {"type": "RandomDataProvider"},
-            "asset": None,
```

Comment:
Ok, maybe my misunderstanding is here, because this isn't actually the structure of the metadata returned from the builder, because there is no `build-metadata` key here? Should we wait for #795 to get in maybe?

Reply:
Yeah, let's do that. Feels a bit unnecessary to fix this.

Reply:
Ok, this one is resolved now :+1: